.circleci/config.yml

```diff
@@ -199,6 +201,12 @@ workflows:
           requires:
             - image-build
           filters: *tagged_build_filters
+      - slack/approval-notification:
+          webhook: ${SLACK_WEBHOOK}
+          message: "Deploy pending approval: https://circleci.com/workflow-run/$CIRCLE_WORKFLOW_ID"
+          channel: "CG2K78UH2" # Optional: If set, overrides webhook's default channel setting
```

Comment:
Can we hide `channel` ID? We can add it to the list of env variables as `SLACK_CHANNEL`

Reply:
@smusali do I need to have the channel id here at all? Just curious.

Reply:
@emily2005, not sure but you are using `channel: "CG2K78UH2"` here as `channel: "<channel ID>"`, right? And it's working; so, just hiding is gonna be enough.

Reply:
@emily2005, I still see the channel ID explicitly here - can we use environment variable here like `SLACK_CHANNEL`? CC: @gregswift

Reply:
@smusali i added the variable in circleci and replaced it in the code. Please let me know if this is ok!